explore/src/main/scala/explore/targeteditor/SearchForm.scala

```diff
@@ -119,6 +123,11 @@ object SearchForm {
           disabled = disabled,
           errorClazz = ExploreStyles.InputErrorTooltipBelow,
           errorPointing = LabelPointing.Above,
+          onBlur = (r: ValidatedNec[NonEmptyString, NonEmptyString]) =>
+            r match {
+              case Valid(name)   => props.targetView.zoom(Target.Sidereal.name).set(name)
+              case Invalid(name) => Callback.log(name)
+            },
```

Comment:
Could this be done with a `withOnMod` on the view being used for the `value`?

Reply:
I don't follow...

Reply:
In the props we don't have a `View[NonEmptyString]` of the name if we wanted to set it, only the concrete name

Reply:
I was thinking of something like `value = View.fromState($).zoom(State.searchTerm).withOnMod(name => props.targetView.zoom(Target.Sidereal.name).set(name))`. Or something like that. I think it does the same thing as you have here. IIRC, the view is only changed on Valid so you don't need to worry about that. But maybe it isn't any better?

Reply:
I see, that's right  we could that